### PR TITLE
gitignore: Add vagrant.kubeconfig file

### PR DESCRIPTION
**Pull request by vadorovsky**
Signed-off-by: Michal Rostecki <mrostecki@suse.de>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/5474)

### CODE REVIEW

**Comment by vadorovsky** [comment]
test-me-please

**Review by aanm — APPROVED** [review]

**Review by ianvernon — APPROVED** [review]

**Comment by ianvernon** [comment]
The change is self-evident; merging without waiting for janitor approval.

**Pull request merged**
